client/src/visualiser-debugger/Component/Control/Controls.tsx

```diff
@@ -14,7 +14,7 @@ import { isInitialBackendState } from '../../Types/backendType';
 import { DEFAULT_MESSAGE_DURATION, useToastStateStore } from '../../Store/toastStateStore';
 
 const BUFFER_THRESHOLD = 30;
-const Controls = () => {
+const Controls = ({ onboardingCompile }: { onboardingCompile: () => void }) => {
```

Comment:
We can import **onboardingCompile** from **_Store/onboardingStore_**. There's no need for create extra dependency.